config/ember-try.js

```diff
@@ -1,54 +1,5 @@
+/* eslint-env node */
 module.exports = {
-  scenarios: [
-    {
-      name: 'default',
-      dependencies: { }
-    },
-    {
-      name: 'ember-release',
-      dependencies: {
-        'ember': 'components/ember#release',
-        'ember-data': 'components/ember-data#release'
-      },
-      resolutions: {
-        'ember': 'release',
-        'ember-data': 'beta'
-      }
-    },
-    {
-      name: 'ember-earliest',
-      dependencies: {
-        'ember': 'components/ember#1.12',
-        'ember-data': 'components/ember-data#1.13',
-        'ember-cli-shims': '0.0.6'
-      },
-      resolutions: {
-        'ember': '1.12',
-        'ember-data': '1.13',
-        'ember-cli-shims': '0.0.6'
-      }
-    },
-    {
-      name: 'ember-beta',
-      dependencies: {
-        'ember': 'components/ember#beta',
-        'ember-data': 'components/ember-data#beta'
-      },
-      resolutions: {
-        'ember': 'beta',
-        'ember-data': 'beta'
-      }
-    },
-    {
-      name: 'ember-canary',
-      dependencies: {
-        'ember': 'components/ember#canary',
-        'ember-data': 'components/ember-data#canary'
-      },
-      resolutions: {
-        'ember': 'canary',
-        'ember-data': 'canary'
-      }
-    }
-  ]
-};
+  command: 'npm test',
```

Comment:
this seems recursive. `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call `ember try:each` will call `npm test` will call ...